sys/include/net/ng_ipv6/addr.h

```diff
@@ -47,6 +47,15 @@ extern "C" {
                                             "ffff:ffff:255.255.255.255"))
 
 /**
+ * @brief The first 10 bits of a site-local IPv6 unicast address
+ *
+ * @see <a href="http://tools.ietf.org/html/rfc4291#section-2.5.7">
+ *          RFC 4291, section 2.5.7
+ *      </a>
+ */
+#define NG_IPV6_ADDR_SITE_LOCAL_PREFIX  (0xFEC0)
```

Comment:
Right. But maybe we should mention somewhere, that this type is deprecated and should not be used (+ a link to [RFC 3879](http://tools.ietf.org/html/rfc3879) but can't be removed for this reason.